spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -194,6 +194,47 @@
     end
   end
 
+  describe 'alphaomegaalpha list' do
+    let(:record) do
+      Traject::NokogiriReader.new(
+        File.read(
+          Arclight::Engine.root.join('spec', 'fixtures', 'ead', 'nlm', 'alphaomegaalpha.xml')
+        ).to_s,
+        {}
+      ).to_a.first
+    end
+
+    it 'selects the components' do
+      expect(result['components'].length).to eq 37
+    end
+
+    context 'when nested component' do
+      let(:self_access_restrict_component) { result['components'].find { |c| c['ref_ssi'] == ['aspace_dba76dab6f750f31aa5fc73e5402e71d'] } }
```

Comment:
These variable names might be more clear if they were like 'component_with_access_restrict', 'component_inheriting_access_restrict', or along those lines.